samples/matter/light_bulb/README.rst

```diff
@@ -73,33 +73,43 @@ Configuration
 
 |config|
 
+
+.. _matter_light_bulb_build_types:
+
 Matter light bulb build types
 =============================
 
 .. matter_light_bulb_sample_configuration_file_types_start
 
-The sample uses different configuration files depending on the supported features.
-Configuration files are provided for different build types and they are located in the application root directory.
-
-The :file:`prj.conf` file represents a ``debug`` build type.
-Other build types are covered by dedicated files with the build type added as a suffix to the ``prj`` part, as per the following list.
-For example, the ``release`` build type file name is :file:`prj_release.conf`.
-If a board has other configuration files, for example associated with partition layout or child image configuration, these follow the same pattern.
-
-.. include:: /config_and_build/modifying.rst
-   :start-after: build_types_overview_start
-   :end-before: build_types_overview_end
-
-Before you start testing the application, you can select one of the build types supported by the sample.
-This sample supports the following build types, depending on the selected board:
-
-* ``debug`` -- Debug version of the application - can be used to enable additional features for verifying the application behavior, such as logs or command-line shell.
-* ``release`` -- Release version of the application - can be used to enable only the necessary application functionalities to optimize its performance.
-* ``no_dfu`` -- Debug version of the application without Device Firmware Upgrade feature support - can be used for the nRF52840 DK, nRF5340 DK, nRF7002 DK, and nRF21540 DK.
-
-.. note::
-    `Selecting a build type`_ is optional.
-    The ``debug`` build type is used by default if no build type is explicitly selected.
+The sample does not use a single :file:`prj.conf` file.
+Configuration files are provided for different build types, and they are located in the sample root directory.
+Before you start testing the application, you can select one of the build types supported by the application, depending on the building method.
+
+See :ref:`app_build_additions_build_types` and :ref:`modifying_build_types` for more information about this feature of the |NCS|.
+
+The sample supports the following build types:
+
+.. list-table:: Sample build types
+   :widths: auto
+   :header-rows: 1
+
+   * - Build type
+     - File name
+     - Supported board
+     - Description
+   * - Debug
+     - :file:`prj.conf`
+     - All from `Requirements`_
+     - | Debug version of the application; can be used to enable additional features for verifying the application behavior, such as logs or command-line shell.
+       | Used by default if no build type is explicitly selected.
```

Comment:
```suggestion
     - Debug version of the application; can be used to enable additional features for verifying the application behavior, such as logs or command-line shell.

       Used by default if no build type is explicitly selected.
```
Forced line break seems to be increasing line spacing:
![image](https://github.com/nrfconnect/sdk-nrf/assets/83582261/6b2d2aab-6fda-4978-a215-c39226404576)

Reply:
Moved info about default to a different column.